cli/lib/util.js

```diff
@@ -474,15 +474,6 @@ const util = {
 
       if (osArch === 'arm64') return 'arm64'
 
-      if (osPlatform === 'darwin') {
-        // could possibly be x64 node on arm64 darwin, check if we are being translated by Rosetta
-        // https://stackoverflow.com/a/65347893/3474615
-        const { stdout } = await execa('sysctl', ['-n', 'sysctl.proc_translated']).catch(() => '')
-
-        debug('rosetta check result: %o', { stdout })
-        if (stdout === '1') return 'arm64'
-      }
-
```

Comment:
Compare removed logic to `arch` logic handling:

https://github.com/feross/arch/commit/6a6c1f9e9d7aef0e5e6a8662ff209795a0baf7e6#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L22